python/python/benchmarks/test_search.py

```diff
@@ -208,14 +220,23 @@ def test_filtered_search(test_dataset, benchmark, selectivity, prefilter, use_in
         "filterable < 5000",
         "filterable > 5000",
     ),
+    ids=[
+        "none",
+        "equality",
+        "not_equality",
+        "in_list_one",
+        "not_in_list_one",
+        "not_equality_and_chain",
+        "in_list_three",
+        "less_than_selective",
+        "greater_than_not_selective",
+    ],
```

Comment:
Otherwise pytest uses the filter expression as part of the id and then I can't select a single test using `-k test_name` because `test_name` has spaces in it and I couldn't figure out the syntax to make that work.

Reply:
On mac, I just wrap in single quotes and it works. I already have to do that in my shell because of `[`.

Reply:
I tried single quotes (I also have to do that for the `[` but it didn't like the test I was working with.  Maybe it was the `=` instead of the ` `?